src/mon/Monitor.cc

```diff
@@ -23,6 +23,7 @@
 #include <limits.h>
 #include <cstring>
 #include <boost/scope_exit.hpp>
+#include <boost/make_shared.hpp>
```

Comment:
Do we need to add this header, or remove it in favor of the Standard Library's <memory> header?

Reply:
We need to add it. This file was depending on it being pulled in by `any.h`.